src/app/shared/form-field/form-field.ts

```diff
@@ -17,7 +18,8 @@ export class TextFormField extends FormField {
        public label: string, 
        public defaultValue: string = '',
        public placeholder: string = '',
-       public validations: Validator[] = []) {
+       public validations: Validator[] = [],
+       public tooltip: Tooltip = new Tooltip()) {
```

Comment:
Then, I guess something like this will be fine and could also work:
`public tooltip: Tooltip | Object = {}` and empty `{}` will be assigned by default to those which don't have tooltips defined by form-field.service.

Let me know what do you think about ^this?